examples/storm-surge/ike/setplot.py

```diff
@@ -161,10 +162,10 @@ def gulf_after_axes(cd):
     plotaxes.ylimits = gulf_ylimits
     plotaxes.afteraxes = gulf_after_axes
 
-    surge.add_surface_elevation(plotaxes, plot_type='contourf', 
+    surge.plot.add_surface_elevation(plotaxes, plot_type='contourf',
```

Comment:
This line is wrong -- should be `surgeplot.add_surface_elevation(...`

@mandli: not sure if this is wrong in other branches.